config/config.go

```diff
@@ -101,52 +92,45 @@ func ReadConfig(configPath string) (*Config, error) {
-		TCRootProgramUserProgramDaemon:  LoadOptionalConfigBool(confReader, "tc-root-program", "user-program-daemon", false),
-		EBPFChainDebugAddr:              LoadOptionalConfigString(confReader, "ebpf-chain-debug", "addr", "0.0.0.0:8899"),
-		EBPFChainDebugEnabled:           LoadOptionalConfigBool(confReader, "ebpf-chain-debug", "enabled", false),
-		L3afConfigsRestAPIAddr:          LoadOptionalConfigString(confReader, "l3af-configs", "restapi-addr", "localhost:53000"),
-		L3afConfigStoreFileName:         LoadOptionalConfigString(confReader, "l3af-config-store", "filename", "/etc/l3afd/l3af-config.json"),
-		MTLSEnabled:                     LoadOptionalConfigBool(confReader, "mtls", "enabled", true),
-		MTLSMinVersion:                  minTLSVersion,
-		MTLSCertDir:                     LoadOptionalConfigString(confReader, "mtls", "cert-dir", "/etc/l3afd/certs"),
-		MTLSCACertFilename:              LoadOptionalConfigString(confReader, "mtls", "cacert-filename", "ca.pem"),
-		MTLSServerCertFilename:          LoadOptionalConfigString(confReader, "mtls", "server-cert-filename", "server.crt"),
-		MTLSServerKeyFilename:           LoadOptionalConfigString(confReader, "mtls", "server-key-filename", "server.key"),
-		MTLSCertExpiryWarningDays:       LoadOptionalConfigInt(confReader, "mtls", "cert-expiry-warning-days", 30),
+		PIDFilename:                 LoadConfigString(confReader, "l3afd", "pid-file"),
+		DataCenter:                  LoadConfigString(confReader, "l3afd", "datacenter"),
+		BPFDir:                      LoadConfigString(confReader, "l3afd", "bpf-dir"),
+		BPFLogDir:                   LoadConfigString(confReader, "l3afd", "bpf-log-dir"),
+		MinKernelMajorVer:           LoadConfigInt(confReader, "l3afd", "kernel-major-version"),
+		MinKernelMinorVer:           LoadConfigInt(confReader, "l3afd", "kernel-minor-version"),
+		KFRepoURL:                   LoadConfigString(confReader, "kf-repo", "url"),
+		HttpClientTimeout:           LoadConfigDuration(confReader, "l3afd", "http-client-timeout"),
+		MaxNFReStartCount:           LoadConfigInt(confReader, "l3afd", "max-nf-restart-count"),
+		BpfChainingEnabled:          LoadOptionalConfigBool(confReader, "l3afd", "bpf-chaining-enabled", true),
+		MetricsAddr:                 LoadConfigString(confReader, "web", "metrics-addr"),
+		KFPollInterval:              LoadOptionalConfigDuration(confReader, "web", "kf-poll-interval", 30*time.Second),
+		NMetricSamples:              LoadOptionalConfigInt(confReader, "web", "n-metric-samples", 20),
+		ShutdownTimeout:             LoadConfigDuration(confReader, "l3afd", "shutdown-timeout"),
+		SwaggerApiEnabled:           LoadOptionalConfigBool(confReader, "l3afd", "swagger-api-enabled", false),
+		Environment:                 LoadOptionalConfigString(confReader, "l3afd", "environment", ENV_PROD),
+		BpfMapDefaultPath:           LoadOptionalConfigString(confReader, "l3afd", "BpfMapDefaultPath", "/sys/fs/bpf"),
```

Comment:
I think this should be required field in config with no default and docs give example like /sys/fs/bpf.